cala-ledger/src/account/mod.rs

```diff
@@ -102,6 +120,24 @@ impl Accounts {
             .await?;
         Ok(())
     }
+
+    #[cfg(feature = "import")]
+    pub async fn sync_account_update(
+        &self,
+        mut db: sqlx::Transaction<'_, sqlx::Postgres>,
+        recorded_at: DateTime<Utc>,
+        origin: DataSourceId,
+        values: AccountValues,
+    ) -> Result<(), AccountError> {
+        let mut account = Account::import(origin, values);
```

Comment:
You can’t import. You need to load update persist. Import would create a new entity

Reply:
From<Vec<String>, AccountValues> for AccountUpdate {
}

let entity = repo.find_imported(id, data_source)?;

entity.update((fields, values): impl Into<AccountUpdate>) {
}
repo.persist(entity)